flutter/documentation/result-spec.md

```diff
@@ -114,36 +114,65 @@ If you enable Submission mode, both `performance_run` and `accuracy_run` values
 
 Info about environment the app is running in. May change when you update your OS, change device hardware, or use another device.
 
-* `os`: string enum  
+* `device_type`: string
```

Comment:
> Would `platform` instead of `device_type` more suitable here?

Yeah, maybe. Sounds good to me, I'll change it.

> What would be all possible values?

I was thinking that maybe it would be beneficial to have a separate field for windows_arm devices, if they need some special handling (for example, full_cpu_name field may not work on ARM, and we would need to do something like `board` on android).
Maybe a separate type for windows laptops, since thay can possibly have a defined manufacturer and model.

Reply:
done